libs/hls.liq

```diff
@@ -18,7 +19,7 @@ def input.hls.native(~id="",~reload=10.,uri)
 
       m = string.extract(pattern="#EXT-X-MEDIA-SEQUENCE:(\\d+)",file.contents(pl))
       pl_sequence = m[1]
-      log.info(label=id,"Sequence: " ^ pl_sequence)
+      log.info(label=label_id,"Sequence: " ^ pl_sequence)
```

Comment:
```suggestion
      log.info(label=id,"Sequence: " ^ pl_sequence)
```

Reply:
We need the name change, `id` is now `string?` but `label_id` is `string`.